static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -39,6 +41,11 @@ define([
         setProgressTracker(atomId);
         tracking.track('play', getTrackingId(atomId));
 
+        var mainContainer = document.querySelectorAll('.immersive-main-media .youtube-media-atom');
+        if (mainContainer.length>0) {
+            document.querySelector('.immersive-main-media').classList.add('atom-playing');
+        }
+
```

Comment:
There's no need to declare the variable mainContainer as it's only used once in the following if condition. I'd write the if condition as...

if ( document.querySelector('.immersive-main-media .youtube-media-atom')) {...}

I've used querySelector not querySelectorAll as then you don't have to check the length, if querySelector doesn't find the element it returns null which is falsy.